config/sync/field.field.node.agency_synonym.body.yml

```diff
@@ -0,0 +1,23 @@
+uuid: a5f311f3-f5c0-4dc2-8e22-40a1cf241db4
```

Comment:
I think this is the file that needs to go away in order to remove the "body" field. Here's my process: 

1. I updated with your latest code and noticed the "Body" field was still visible in the CMS. 
2. verified that my config was synchronized. 
3. In the CMS, edited the agency_synonym, deleting the "Body" field.
4. exported config (full export) 
5. deleted all the .yml files from my config/sync directory
6. moved the export file config-[...]-[date].tar.gz into config/sync, and extracted it. (Then I delete the export file.) 
7. Use "git status" to show what changed. 

All I see is that this file was deleted. 

I go through these steps, or something close, just about every time I commit config changes. There are other approaches, but this works well for me.

Reply:
Thank you! That showed the deleted file in my changes too. I was also exporting the full config but i was just dragging the new files into the old directory and replacing all. Because of that the unused file was still in it.